content/microservices/grpc.md

```diff
@@ -233,6 +233,16 @@ imports: [
 
 > info **Hint** The `register()` method takes an array of objects. Register multiple packages by providing a comma separated list of registration objects.
 
+Later in the `nest-cli.json` file, we add the assets property that allows us to distribute non-TypeScript files, in our case the `.proto` file. This way, during the build phase, you can edit non-TypeScript files and compile and test iteratively.
+
+```json
+{
+  "compilerOptions": {
+    "assets": ["**/*.proto"]
+  }
+}
+```
+
```

Comment:
This addition looks great overall, but I don't that "Client" heading is an appropriate section for this (especially in the middle of the `ClientsModule` example which is completely unrelated to this newly inserted block).

Reply:
What if we put it before the subtitle "Client"?

Reply:
I think the best place to add it would be the "Overview" section (by the end of it) @Tony133

Reply:
Moved to the "Overview" section as you said